doc/modules/mod_keystore.md

```diff
@@ -20,18 +20,19 @@ for each virtual XMPP domain to avoid sharing keys between domains!**
 
 ### Options
 
-* `ram_key_size`: size to use when generating RAM-only keys (designated
-    by type `ram`)
-* `keys`: list of _specifiers_ of keys which will be provided by the
-    module at runtime
+#### `modules.mod_keystore.ram_key_size`
+* **Syntax:** non-negative integer
+* **Default:** `1000`
+* **Example:** `ram_key_size = 10000`
 
-Each _key specifier_ is a pair of `{KeyName, KeyType}`, where:
+Size to use when generating RAM-only keys (designated by type `ram`).
 
-* `KeyName`: any Erlang term. For simplicity's sake atoms are advised.
-    Names have to be unique in the context of one virtual domain.
-* `KeyType`: one of `ram` or `{file, "path/to/file"}`.
-    The file is read and its contents are provided
-    as the key (whitespace is trimmed).
+#### `modules.mod_keystore.keys`
+* **Syntax:** Array of TOML tables with the following keys: `"name"`, `"type"`, `"file"`, and following values: {name = `string`, type = `values: "file", "ram"`, file = `string`}.
+* **Default:** `[name = "", type = ""]`
```

Comment:
Is this default correct? Considering that `path` is used only when `type = "file"`.